api/core/agent/base_agent_runner.py

```diff
@@ -492,7 +493,8 @@ def organize_agent_history(self, prompt_messages: list[PromptMessage]) -> list[P
         return result
 
     def organize_agent_user_prompt(self, message: Message) -> UserPromptMessage:
-        files = db.session.query(MessageFile).where(MessageFile.message_id == message.id).all()
+        stmt = select(MessageFile).where(MessageFile.message_id == message.id)
+        files = db.session.execute(stmt).scalars().all()
```

Comment:
I guess

```suggestion
        files = db.session.scalars(stmt).all()
```

ref https://docs.sqlalchemy.org/en/20/orm/queryguide/select.html#orm-queryguide-select-orm-entities

Reply:
Thanks for your suggestion — this way is both concise and correct.
Ref: 531008d